src/annmas/annotate/command.py

```diff
@@ -109,14 +110,23 @@ def main(model, threads, output_bam, input_bam):
     # input_data_queue = ManyProducerSingleConsumerIpcQueue()
 
     for i in range(threads):
-        p = mp.Process(target=_worker_segmentation_fn, args=(input_data_queue, results, i, m))
+        p = mp.Process(
+            target=_worker_segmentation_fn, args=(input_data_queue, results, i, m)
+        )
         p.start()
         worker_pool.append(p)
 
     pysam.set_verbosity(0)  # silence message about the .bai file not being found
     with pysam.AlignmentFile(
         input_bam, "rb", check_sq=False, require_index=False
-    ) as bam_file, tqdm.tqdm(desc="Progress", unit=" read", colour="green", file=sys.stdout) as pbar:
+    ) as bam_file, tqdm.tqdm(
+        desc="Progress",
+        unit=" read",
+        colour="green",
+        file=sys.stderr,
+        leave=False,
```

Comment:
This option erases the progress meter from the screen when processing is complete.  To be honest, I don't love this.  However, without it, there's a bit of a weird formatting issue when the iterations complete.  It's almost like the iterations are being reset to 0 and printed to the screen.  Like so:

```
$ python -m annmas annotate -t 0 -o /dev/null ../scratch/test_input.bam
[INFO 2021-02-06 00:06:58 annotate] Invoked via: annmas annotate -t 0 -o /dev/null ../scratch/test_input.bam
[INFO 2021-02-06 00:06:58 annotate] Running with 12 worker subprocess(es)
[INFO 2021-02-06 00:07:01 annotate] Using default annotation model
Progress: 0 read [00:10, ? read/s]ad/s]
[INFO 2021-02-06 00:07:11 annotate] Annotated 100 reads with 7203 total sections.
[INFO 2021-02-06 00:07:11 annotate] Done. Elapsed time: 12.53s.
```

Hiding the progress meter on completion fixes this, but then we lose the really nice read processing rate at the end.  So if you know how to make this look nice without tearing the progress meter down, I'm all for it.